apps/nextjs/tests-e2e/tests/aila-chat/full-romans.test.ts

```diff
@@ -55,14 +57,13 @@ test(
     await test.step("Iterate through the fixtures", async () => {
       await page.waitForURL(/\/aila\/.+/);
 
-      const maxIterations = 20;
+      const maxIterations = 10;
 
       for (
-        let iterationCount = 1;
+        let iterationCount = 2;
```

Comment:
We've already triggered a response from Aila in the initial step and it is called "1". This was causing it to reuse the initial fixture for the next "continue", and because we now try to pick the right section group, it was ending up in some strange behaviour.

Reply:
Oh, that's a great spot